package org.example.exception.exceptions

class UsingReasonUnableException(): RuntimeException("입력하신 이용사유는 존재하지 않습니다") {
}